### PR TITLE
CU-24vyq5d_Fix-issues-with-prettier_Mateo-Sanchez

### PR DESCRIPTION
**Pull request by mallow43**
Created prettierrc.json and removed alphabetization of import keys from eslintrc.js

### CODE REVIEW

**Comment by mallow43** [comment]
Task linked: [CU-24vyq5d Fix issues with prettier](https://app.clickup.com/t/24vyq5d)

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "semi": true,
+    "trailingComma": "none",
```
Comment (j-mateo) [inline .prettierrc.json:3]: just out of curiosity, why did we decide against trailing commas?

**Review by j-mateo — APPROVED** [review]
lgtm, just a few questions
Also, are you planning to put up another PR to apply the new format to existing files, or just let them get fixed organically as they get edited?

File: .prettierrc.json (hunk 1 above)
Comment (HenryQuillin) [reply .prettierrc.json:3]: Not sure. Personally, I'm all for trailing commas. It makes adding properties much easier.

Comment (scandycuz) [reply .prettierrc.json:3]: I'm in favor of trailing commas as well

File: .eslintrc.js [hunk 2]
```diff
@@ -45,10 +45,10 @@ module.exports = {
     "import/order": [
       "warn",
       {
-        alphabetize: {
-          caseInsensitive: true,
-          order: "asc",
-        },
+        // alphabetize: {
```
[the remark this answers is not in the document]
Comment (scandycuz) [reply .eslintrc.js:48]: I agree we should just remove this code instead of commenting it out

**Review by scandycuz — DISMISSED** [review]
I think we should make the suggested changes before merging it.

**Comment by mallow43** [comment]
@j-mateo I was having trouble replying to your comment. But yeah I was just hoping to have it happen organically because if not I was thinking everyones working code will have a ton of merge conflicts. But maybe there is actually a way to do it so let me know.

**Pull request merged**